src/pages/workspaces/workspace/Data.js

```diff
@@ -666,6 +683,16 @@ const WorkspaceData = _.flow(
     setEntityMetadata(updatedMetadata)
   }
 
+  const searchAcrossTables = async (types, activeCrossTableTextFilter) => {
+    setCrossTableSearchInProgress(true)
+    const results = await Promise.all(_.map(async ([type]) => {
```

Comment:
I was confused about why the async function's argument was a destructured array. Should `searchAcrossTables` take a list of type names instead of the current list of `[typeName, typeInfo]`?

Reply:
That would make more sense!

Reply:
Addressed in d068abed0668ed29db995c885beff1f431cce470